### ISSUE TITLE
Keyboard navigation does not work when browsing a channel (Firefox)

### ISSUE THREAD

**Issue by sairina** [learningequality/kolibri#9191]
## Observed behavior
When a user is inside a channel (within a topic), keyboard navigation does not work.

## Errors and logs
None

## Expected behavior
That keyboard navigation works

## User-facing consequences
Cannot use keyboard navigation inside a channel.

## Steps to reproduce
1. Sign in as learner
2. Use the mouse to navigate to the library, and click on a channel
3. Use the keyboard to attempt to navigate. Notice that there is no visual indicator that you can navigate through the content

## Context
Kolibri 0.15
Firefox

**Comment by marcellamaki** [comment]
Possible mac settings issue 

![in-topic-nav-firefox.gif](https://images.zenhubusercontent.com/6047f6e691103a78d3ad0ace/7a774e3c-b1c2-4477-814e-24a6540299ba)

**Comment by marcellamaki** [comment]
cc @radinamatic -- to double check before closing

**Comment by pcenov** [comment]
Confirmed - with the correct settings enabled the tab navigation works correctly on Safari, Firefox and Chrome on Mac.
